pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [project]
-requires-python = ">=3.7"
+requires-python = ">=3.8"
```

Comment:
pure fyi: the reason we deprecated 3.7 support was because we used `ubuntu-latest` in CI and when Github Actions bumped latest == 22.04 we lost python 3.7 there. While we could have patched it in from the `deadsnakes` PPA we thought that silly.

And, since we shouldn't claim support for that which we don't (or can't) test, then it is proper for us to declare 3.8+

Reply:
I'd like to add that support for 3.7 ends in 6 months as well.